openjdk.build/build.xml

```diff
@@ -55,19 +51,13 @@ limitations under the License.
 		<ant antfile="${source_root}/openjdk.test.serialization/build.xml" dir="${source_root}/openjdk.test.serialization" inheritAll="false"/>
 		<ant antfile="${source_root}/openjdk.test.util/build.xml" dir="${source_root}/openjdk.test.util" inheritAll="false"/>
 		<ant antfile="${source_root}/openjdk.test.jlm/build.xml" dir="${source_root}/openjdk.test.jlm" inheritAll="false"/>
+		<ant antfile="${source_root}/openjdk.test.modularity/build.xml" dir="${source_root}/openjdk.test.modularity" inheritAll="false"/>
```

Comment:
Will the source be built by any jdk version or build.xml has flag to do condition compiling?

Reply:
Good catch! It should not be built for Java 8, it can be built for Jaca 9 and up. Need to add a check here. WIll update the PR.

Reply:
@sophia-guo  - a check has been added.